### PR TITLE
Update installing-the-github-copilot-extension-in-your-environment.md

### PR DESCRIPTION
**Pull request by sammyhori**
### Why:

- If someone was installing copilot on Neovim, they would reasonably come across the vim section, follow the instructions and then realise, as they scroll down, they did it incorrectly,
- On the other hand, if someone was installing it on vim, they would be much less likely to read the Neovim part and believe it was for them.

Also, the two sections are very difficult to tell apart when you first come across them, due to the differences in them being minor.

Closes: 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Switched the Neovim and Vim install instructions around.

To further emphasise the difference between the sections, I put vim and neovim in italics in the subheadings.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@sammyhori Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/copilot/managing-copilot/configure-personal-settings/installing-the-github-copilot-extension-in-your-environment.md [hunk 1]
```diff
@@ -100,32 +100,32 @@ To see instructions for other popular coding environments, use the tool switcher
 
 1. Install {% data variables.product.prodname_copilot %} using the built-in plugin manager:
 
-   * For **Vim on macOS or Linux**, run the following command in the terminal.
+   * For **_Neovim_ on macOS or Linux**, run the following command in the terminal.
```
Comment (jules-p) [inline content/copilot/managing-copilot/configure-personal-settings/installing-the-github-copilot-extension-in-your-environment.md:103]:
```suggestion
   * For **Neovim on macOS or Linux**, run the following command in the terminal.
```

File: content/copilot/managing-copilot/configure-personal-settings/installing-the-github-copilot-extension-in-your-environment.md [hunk 2]
```diff
@@ -100,32 +100,32 @@ To see instructions for other popular coding environments, use the tool switcher
...
 
      ```shell copy
      git clone https://github.com/github/copilot.vim \
-     ~/.vim/pack/github/start/copilot.vim
+     ~/.config/nvim/pack/github/start/copilot.vim
      ```
 
-   * For **Vim on Windows**, run the following command in Git Bash:
+   * For **_Neovim_ on Windows**, run the following command in Git Bash:
```
Comment (jules-p) [inline content/copilot/managing-copilot/configure-personal-settings/installing-the-github-copilot-extension-in-your-environment.md:110]:
```suggestion
   * For **Neovim on Windows**, run the following command in Git Bash:
```

File: content/copilot/managing-copilot/configure-personal-settings/installing-the-github-copilot-extension-in-your-environment.md [hunk 3]
```diff
@@ -100,32 +100,32 @@ To see instructions for other popular coding environments, use the tool switcher
...
 
      ```shell copy
      git clone https://github.com/github/copilot.vim.git \
-     $HOME/vimfiles/pack/github/start/copilot.vim
+     $HOME/AppData/Local/nvim/pack/github/start/copilot.vim
      ```
 
-   * For **Neovim on macOS or Linux**, run the following command in the terminal.
+   * For **_Vim_ on macOS or Linux**, run the following command in the terminal.
```
Comment (jules-p) [inline content/copilot/managing-copilot/configure-personal-settings/installing-the-github-copilot-extension-in-your-environment.md:117]:
```suggestion
   * For **Vim on macOS or Linux**, run the following command in the terminal.
```

File: content/copilot/managing-copilot/configure-personal-settings/installing-the-github-copilot-extension-in-your-environment.md [hunk 4]
```diff
@@ -100,32 +100,32 @@ To see instructions for other popular coding environments, use the tool switcher
...
 
      ```shell copy
      git clone https://github.com/github/copilot.vim \
-     ~/.config/nvim/pack/github/start/copilot.vim
+     ~/.vim/pack/github/start/copilot.vim
      ```
 
-   * For **Neovim on Windows**, run the following command in Git Bash:
+   * For **_Vim_ on Windows**, run the following command in Git Bash:
```
Comment (jules-p) [inline content/copilot/managing-copilot/configure-personal-settings/installing-the-github-copilot-extension-in-your-environment.md:124]:
```suggestion
   * For **Vim on Windows**, run the following command in Git Bash:
```

**Review by jules-p — DISMISSED** [review]
Hi @sammyhori :wave: thank you for taking the time to contribute to our docs, we really appreciate it. The combination of italics and bold goes against our [style guide](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#emphasis), so I've removed that, but I'll move forward with the reordering as you suggested. Thank you again :sparkles:

**Review by jules-p — APPROVED** [review]

**Pull request merged**